db/migrate/20170713021445_create_invites.rb

```diff
@@ -0,0 +1,14 @@
+class CreateInvites < ActiveRecord::Migration[5.1]
+  def change
+    enable_extension 'pgcrypto'
+
+    create_table :invites, id: :uuid do |t|
+      t.string :email
+      t.bigint :inviter_id
+      t.bigint :course_id
```

Comment:
this can be changed to `t.references :course`

`inviter_id` is different because it's unconventional